src/main/resources/application.conf

```diff
@@ -36,6 +36,12 @@ workflow-options {
 
   // AES-256 key to use to encrypt the values in `encrypted-fields`
   base64-encryption-key: "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="
+
+  // Directory where to write per workflow logs
+  workflow-log-dir: "cromwell-workflow-logs"
+
+  // When true, per workflow logs will be deleted after copying
+  workflow-log-temporary: true
```

Comment:
Now that I think about it I don't actually *know for sure* what the default logging behavior is for other big-name servers.  I just have a feeling (anecdata?) that usually I don't have to turn on file-logging by default.  I'll try to investigate out of curiosity when I have some time today